src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -252,6 +252,16 @@ public static String formatDateTimeForInstructorHomePage(Date date) {
         return sdf.format(date);
     }
 
+    public static String formatDateForFeedbackSubmissionEditPage(Date date, double timeZone) {
+        if (date == null) {
+            return "";
+        }
+        // Passing negative timeZone to convert to UTC.
+        Calendar calInUtc = convertToUserTimeZone(dateToCalendar(date), -timeZone);
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.S'Z'");
```

Comment:
This can be further extracted into a helper method. Extract the string into Constants.